spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/DashboardHelperService.java

```diff
@@ -176,6 +177,41 @@ public JSONObject searchProject(HttpSession session, JSONObject json) {
     }
   }
 
+
+  public JSONObject searchPool(HttpSession session, JSONObject json) {
+    String searchStr = json.getString("str");
+    try {
+      List<Pool> pools;
+      StringBuilder b = new StringBuilder();
+      if (!isStringEmptyOrNull(searchStr)) {
+
+        pools = new ArrayList<Pool>(requestManager.listAllPoolsBySearch(searchStr));
+      } else {
+        pools = new ArrayList<Pool>(requestManager.listAllPoolsWithLimit(50));
+      }
+
+      if (pools.size() > 0) {
+        Collections.sort(pools);
+        Collections.reverse(pools);
+        for (Pool p : pools) {
+          // yuck-o
```

Comment:
If you're looking for something slightly less yucky, @apmasell did something nice [here](https://github.com/TGAC/miso-lims/blob/develop/miso-web/src/main/webapp/scripts/pool_ajax.js#L960).